Tests/WeatherSDKTests/models/ListCitiesInput+Equatable.swift

```diff
@@ -6,6 +6,8 @@ import WeatherSDK
 extension ListCitiesInput: Swift.Equatable {
 
     public static func ==(lhs: ListCitiesInput, rhs: ListCitiesInput) -> Bool {
+        if lhs.nextToken != rhs.nextToken { return false }
+        if lhs.pageSize != rhs.pageSize { return false }
```

Comment:
These are members that have the `@httpQuery` trait and were previously not included in the comparison